index.src.html

```diff
@@ -394,6 +435,233 @@
     </section>
     <section>
       <h2>
+        Synchronizing depth and color video rendering
+      </h2>
+      <p>
+        When rendering, we want to position a color from color video frame to
+        corresponding <a>depth map value</a> or 3D point in space defined by
+        <a>depth map value</a>. We use <a>deprojection distortion
+        coefficients</a> to compensate camera distortion when deprojecting 2D
+        pixel coordinates to 3D space coordinates and <a>projection distortion
+        coefficients</a> in the opposite case, when projecting camera 3D space
+        points to pixels. The algorithm here intents to cover all of the cases.
```

Comment:
The <a>algorithm to map...</a> and <a>algorithm to xxx...</a> are defined in this specifications.